client/message/history.jsx

```diff
@@ -70,7 +70,7 @@ const History = React.createClass({
                                 message_content: message.message_content,
```

Comment:
Shouldn't the key also be converted to an integer here and in the `if` case above?

Reply:
"here"? The line you've commented on has the `message_content` that's generally a string. If you're referring to Object keys those are generally converted automatically from/to strings, so no point in intervening there. But the types should be more clear once we have flow in place.